.github/scripts/releasecli.sh

```diff
@@ -1,9 +1,11 @@
+#!/bin/bash
+GITHUB_SHA=$1
 VERSION=$(cat version)
 BASE_URL="https://console.cloud.google.com/storage/browser/inspr-cli/bin"
 
 release-cli create --name "Release $VERSION" \
         --tag-name $VERSION \
-        --ref "$CI_COMMIT_SHA" \
+        --ref $GITHUB_SHA \
         --assets-link "{\"name\":\"insprcli-linux-arm64\",
                         \"url\":\"$BASE_URL/insprcli-linux-arm64-$VERSION\"}" \
         --assets-link "{\"name\":\"insprcli-linux-amd64\",
```

Comment:
Is does this CLI actually have the same API as gitlab's? I mean, if so, great, but did you check that?

Reply:
As we discussed, we'll be storing the CLI binaries in the GitHub Release